.github/workflows/tests_numpy.yml

```diff
@@ -35,5 +37,39 @@ jobs:
           poetry config virtualenvs.create false
           poetry install --extras "ray" --with dev
 
+      - name: Initialize Julia project  # to keep test times stable
+        run: python -c 'from juliacall import Main as jl'
+
+      - name: Create durations file name
+        id: record_file
+        run: echo "filename=durations_$(date +%s)_$(git rev-parse --short HEAD).txt" >> "$GITHUB_OUTPUT"
+
       - name: Run tests
-        run: python -m pytest tests --backend=numpy -p no:warnings --tb=native
+        run: |
+          set -o pipefail
+          python -m pytest tests --backend=numpy -p no:warnings --tb=native ${{ github.event_name == 'push' && '--durations=0 -vv ' || '' }}| tee durations.txt
+          set +o pipefail
+
+      # - name: Reformat durations file
+      #   if: github.event_name == 'push'
+      #   run: grep ' call ' durations.txt | awk '{print $3,$1}' > ${{ steps.record_file.outputs.filename }}
+
+      # - name: Upload benchmark results
+      #   if: github.event_name == 'push'
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: timings
+      #     path: ${{ steps.record_file.outputs.filename }}
```

Comment:
if this commented block is not needed let's remove it

Reply:
I'll remove it!